client/src/main/webjar/authentication/authentication.service.js

```diff
@@ -0,0 +1,98 @@
+(function (angular) {
+    "use strict";
+
+    var module = angular.module("healthBam.authentication");
+
+    /**
+     * A service to authenticate
+     */
+    function authenticationServiceFactory(
+        $log,
+        $q,
+        $auth
+    ) {
+
+        /**
+         * Pops open satellizer's login prompt.
+         */
+        function authenticate() {
+            $log.debug("authenticate called");
+
+            return $auth.authenticate("google").then(
+                function (response) {
+                    $log.debug("login succeeded", response);
+                    return response;
+                }
+            ).catch(
+                function (response) {
+                    $log.debug("login failed", response);
+                    return $q.reject(response);
+                }
+            );
+        }
+
+        /**
+         * Returns true if the user has a token and it isn't expired.
+         */
+        function isAuthenticated() {
+            return $auth.isAuthenticated();
+        }
+
+        /**
+         * Returns true if the user is logged in and is an admin.
+         */
+        function isAdmin() {
+            if (isAuthenticated()) {
+                return $auth.getPayload().admin;
```

Comment:
This module/service/ec. just serves as an example.  @mckoon feel free to throw this away and adapt it to your needs.